integration/helpers/deno-template/package.json

```diff
@@ -1,27 +1,28 @@
 {
-  "name": "remix-template-deno",
+  "name": "integration-deno-template",
+  "version": "0.0.0",
   "private": true,
   "sideEffects": false,
   "type": "module",
   "scripts": {
-    "build": "node ../../../build/node_modules/@remix-run/dev/dist/cli.js build",
-    "dev": "node ../../../node_modules/npm-run-all/bin/npm-run-all/index.js build --parallel \"dev:*\"",
-    "dev:deno": "node ../../../node_modules/cross-env/src/bin/cross-env.js NODE_ENV=development deno run --unstable --watch --allow-net --allow-read --allow-env ./build/index.js",
-    "dev:remix": "node ../../../build/node_modules/@remix-run/dev/dist/cli.js watch",
-    "start": "node ../../../node_modules/cross-env/src/bin/cross-env.js NODE_ENV=production deno run --unstable --allow-net --allow-read --allow-env ./build/index.js"
+    "build": "remix build",
+    "dev": "npm-run-all build --parallel \"dev:*\"",
+    "dev:deno": "cross-env NODE_ENV=development deno run --unstable --watch --allow-net --allow-read --allow-env ./build/index.js",
```

Comment:
Also `cross-env` is not needed, you can use the shell emulator https://pnpm.io/cli/run#shell-emulator